### PR TITLE
FEV-184-chapter-side-menu-buttons add correct tabindex and aria label

### PR DESCRIPTION
**Pull request by MaksymKorniiov**

### CODE REVIEW

File: modules/Tvpapi/resources/tvpapiAnalytics.js [hunk 1]
```diff
@@ -5,14 +5,16 @@
 
 		defaultConfig: {
 			mediaHitInterval: 30,
-			startTime:null
+			startTime:null,
```
Comment (eitanavgil) [inline modules/Tvpapi/resources/tvpapiAnalytics.js]: Again - how is this relevant?

Comment (MaksymKorniiov) [reply modules/Tvpapi/resources/tvpapiAnalytics.js]: @eitanavgil this is not my commit

Comment (eitanavgil) [reply modules/Tvpapi/resources/tvpapiAnalytics.js]: @MaksymKorniiov  - I see that now. 
@chanakabr why did you commit this into our branch?
